python-sdk/tests/data/sample.csv

```diff
@@ -1,4 +0,0 @@
-id,name
-1,First
-2,Second
-3,Third with unicode पांचाल
```

Comment:
Why are we deleting the file from python-sdk?  in UTO PR

Reply:
Was not intended, added it back.